.samples/neo42PackageConfigJson.MD

```diff
@@ -558,9 +569,9 @@ switch to control the interpretation of character '*' in a defined string of val
 parent: /
 ```
 default value: false
-switch to control the interpretation of '$($variables)' in a defined string of value "UninstallKey" (valid are only 'true' and 'false')
+switch to control the interpretation of stored variables (such as '$($VariableName)') within the defined string with the value "UninstallKey" (valid are only 'true' and 'false')
```

Comment:
Same as above. Keep simpler.

E.g.
switch to control wether variables in "UninstallKey" are evaluated or kept as text